### PR TITLE
Adding clear method to MockMemcacheClient

### PR DESCRIPTION
**Pull request by FerasAlazzeh**
Adding a `clear` method to `MockMemcacheClient` as a way to reset/clear the mock cache.

### CODE REVIEW

File: pymemcache/test/utils.py [hunk 1]
```diff
@@ -43,6 +43,9 @@ def __init__(self,
         self.ignore_exc = ignore_exc
         self.encoding = encoding
 
+    def clear(self):
+        self._contents.clear()
```
Comment (jparise) [inline pymemcache/test/utils.py]: Let's add a docblock here because this isn't a standard memcache command.

Comment (FerasAlazzeh) [reply pymemcache/test/utils.py]: Sounds good. Let me know if I should add more for ex. (this method is used strictly for testing to increase convenience).

File: pymemcache/test/utils.py [hunk 2]
```diff
@@ -43,6 +43,10 @@ def __init__(self,
         self.ignore_exc = ignore_exc
         self.encoding = encoding
 
+    def clear(self):
+        '''Method used to clear/reset mock cache'''
```
Comment (jparise) [inline pymemcache/test/utils.py]: We use `"` for docblocks elsewhere in the code. Let's stay consistent.

```suggestion
        """Method used to clear/reset mock cache"""
```

**Comment by jparise** [comment]
Thanks!

**Pull request merged**